mpsl/include/mpsl_radio_notification.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2013 - 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @defgroup mpsl_radio_notification_api MPSL Library Radio Notification API
+ * @{
+ *
+ * @brief APIs for the MPSL Radio Notification module.
+ *
+ */
+
```

Comment:
Get rid of double empty lines in all files. Shouldn't be used anywhere.

Reply:
I think I have removed all of them now